ENIGMAsystem/SHELL/Audio_Systems/SDL/Makefile

```diff
@@ -0,0 +1,14 @@
+ifndef PKG-CONFIG
+	PKG-CONFIG := pkg-config
+endif
+
+SOURCES += $(wildcard Audio_Systems/SDL/*.cpp)
+
+ifeq ($(TARGET-PLATFORM), Android)
+  override ANDROID_MAKEFILES += SDL_mixer/Android.mk
+  override ANDROID_SHARED_LIBRARIES += SDL2_mixer
+endif
+
+override CXXFLAGS += $(shell pkg-config --cflags sdl)
+override LDLIBS += $(shell pkg-config --libs sdl2 SDL2_mixer) -lopusfile -lopus -lvorbisfile -lvorbis -logg -lFLAC -lsndfile -lmodplug -lmpg123 -lole32 -lwinmm -lm -lshlwapi -lssp
```

Comment:
Let's try to clean this up as much as we can. We should not be passing `sdl` or `sdl2` here since that is already handled by the SDL Platform's Makefile, which this system will depend upon anyway. It seems `SDL2_mixer` should also be specified for `CXXFLAGS` too. Finally, please compare the output of `pkg-config --libs SDL2_mixer` to those extra libs, and remove any that are already specified.